helix-vte/src/error.rs

```diff
@@ -0,0 +1,11 @@
+#[derive(Debug, thiserror::Error)]
+pub enum Error {
+    #[error("Pty Error: {0}")]
+    PtyError(#[from] pty_process::Error),
+
+    #[error("IO Error: {0}")]
+    IoError(#[from] std::io::Error),
+
+    #[error("Termina Not Found: {0}")]
```

Comment:
```suggestion
    #[error("Terminal Not Found: {0}")]
```

Totally benign typo - just happened to be reading through the code and spotted it.